docs/design.md

```diff
@@ -22,6 +22,24 @@ These are available <span style="font-family: courier new">superclass</span> val
 
 You might find other superclass values in the workbook XML from an existing Tableau connection, but we don't recommend using them. They haven't been tested.
 
+## Assign a plugin version
+
+Plugin-version is a requirement for submitting your connector to the Tableau Connector Gallery, especially with a partner releasing a newer version in Tableau gallery and notifying customers to download an update. 
+You should start your plugin version with `plugin-version='1.0.0'` and increment it based on major or minor changes to your connector. 
+
+## Set your min-tableau-version
```

Comment:
People don't set this, the packager automatically sets this. Letting them know it exists is fine, but anything they put here will be overwritten by the packager.

Reply:
Updated the part about min-tableau-version being set by the packager.